L1.py

```diff
@@ -0,0 +1,34 @@
+from random import *
+from copy import *
+
+def sortowanie_1(lista):

```

Comment:
Proszę nazywaj funkcje i zmienne po angielsku, ponadto - bardzo ważne- nazywaj je tak, by wiadomo było co i jak robią- tu wiadomo tylko że mamy dwie funkcje sortowania, ale bez przyglądnięcia się nie wiadomo jakim sposobem to robią, dużo lepszą byłaby nazwa bubbleSort.